apps/angular/src/app/features/anime/components/anime-table/anime-table.component.ts

```diff
@@ -0,0 +1,36 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { Anime } from '@js-camp/core/models/anime';
+import { BehaviorSubject, switchMap, Observable, tap } from 'rxjs';
+
+import { AnimeService } from '../../../../../core/services/anime.service';
+
+/**
+ * Anime table component.
+ */
+@Component({
+  selector: 'camp-anime-table',
+  templateUrl: './anime-table.component.html',
+  styleUrls: ['./anime-table.component.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class AnimeTableComponent {
+
+  /** Anime. */
+  public readonly animeList$: Observable<readonly Anime[]>;
+
+  /** Whether books are loading or not. */
+  public readonly isAnimeLoading$ = new BehaviorSubject<boolean>(false);
+
+  /** Displayed columns. */
+  public readonly displayedColumns: string[] = ['imageSrc', 'titleEnglish', 'titleJapanese', 'type', 'status', 'airedStart'];
+
+  private readonly animeUpdated$ = new BehaviorSubject<void>(void 0);
+
+  public constructor(animeService: AnimeService) {
+    this.animeList$ = this.animeUpdated$.pipe(
+      tap(() => this.isAnimeLoading$.next(true)),
+      switchMap(() => animeService.getAnimeList()),
+      tap(() => this.isAnimeLoading$.next(false)),
+    );
```

Comment:
It is really strange that you are using `animeUpdated$` here. To get rid of it, make initial value of `isAnimeLoading$` `true`, than set it to `false` after get anime list.
```suggestion
    this.animeList$ = animeService.getAnimeList().pipe(
      tap(() => this.isAnimeLoading$.next(false))
    )
```

Reply:
This code does not work, the variable isLoading remains false. I will remove this variable for now, but I will solve the problem in the future.